Bottom: c77269b7838d9f23fe93c0ef846ca8465f644d12
Top:    ce7049878674e788db78711717df7e1e9921a649
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-10 17:10:53 -0500

Refresh of add-environment-variables-for

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index df2f075c..9ce67a77 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -36,11 +36,11 @@ config_hostdir: '{{ docker_restore_config_base_dir }}/{{ container_name }}'
 # environment passed to the container
 container_env:
   BUILDBOT_DATABASE: 'sqlite:///{{ data_path }}/state.sqlite'
-  BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
-  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_SMTP_HOST: 'relay.dnsexit.com'
   BUILDBOT_SMTP_USER: 'Dean.cox'
   BUILDBOT_SMTP_PASS: 'no5156'
+  BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
+  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
   BUILDBOT_WORKER_NTEL: 'worker-ntel'
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
